docs/troubleshooting_and_faq/installation.md

```diff
@@ -0,0 +1,32 @@
+# Installation Issues
+
+### Management Pack fails to install
+Management Packs can fail to install for a number of reasons, although it is not common. The best way to diagnose what is happening
+is to look in the logs on the main VMware Aria Operations cluster node (not a Cloud Proxy). To access these logs, `ssh` into the cluster
+and navigate to `$ALIVE_BASE/user/log`. Installation issues are logged to the analytics log, which has the form `analytics-<GUID>.log`.
+Generally, a good place to start is to search that log file for the adapter kind key (found in the `manifest.txt` file in the project 
+directory, as the entry in the `adapter_kinds` array).
```

Comment:
Can these logs also be viewed in the Support section of Aria Ops?

Reply:
They can. The interface isn't great, but some people might not be able to SSH.